src/components/entity/ha-entity-picker.ts

```diff
@@ -260,6 +264,21 @@ export class HaEntityPicker extends LitElement {
     }
   }
 
+  private _filterChanged(ev): void {
+    const filterString = ev.detail.value.toLowerCase();
+
+    const filteredStates = this._states.filter(
+      (state) =>
+        fuzzySequentialMatch(filterString, state.entity_id.toLowerCase()) ||
+        fuzzySequentialMatch(
+          filterString,
+          computeStateName(state).toLowerCase()
```

Comment:
I have another PR up that lets you send multiple strings to fuzseq: https://github.com/home-assistant/frontend/pull/7306

I think it'll merge soon if you want to wait for that.

Reply:
Ah very nice. I will wait for that.